src/components/Api.js

```diff
@@ -1,3 +1,145 @@
 export default class Api {
-  
-}
\ No newline at end of file
+  constructor() {
+    this.baseURL = "https://around.nomoreparties.co";
+  }
+
+  getUserInfo () {
+    return fetch(`${this.baseURL}/v1/group-12/users/me`, {
+      headers: {
+        authorization: "397bd50b-9f39-4bee-ad9c-11e69aa20ec4"
+      }
+    })
+    .then(response => {
+      if (!response.ok) {
+        return Promise.reject(`Error: ${response.status}`);
+      }
+
+      return response.json();
+    })
+    .catch(err => {
+      console.error(err);
+    })
```

Comment:
It’s better not to use `catch` in `Api` because you need to `catch` possible errors at the very end of any request and this end is in `index.js` after all blocks `then`. And in real projects blocks `catch` usually open error popups and so on.

Please, remove it from `Api` and add it in `index.js`.